### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Vlad0os13**
- [DEMO LINK](https://Vlad0os13.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Vlad0os13.github.io/layout_product-cards/report/html_report/)

- [x] there 2 BEM blocks `card` and `stars` each in it own file
- [x] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x] SCSS Variables are used for main values and placed in a **separate file**
- [x] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html:19]:
```suggestion
    <div class="card" data-qa="card">
```

File: src/index.html [hunk 2]
```diff
@@ -13,6 +13,40 @@
...
+      <div class="card__image"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:20]: Add image with the img tag

File: src/index.html [hunk 3]
```diff
@@ -13,6 +13,40 @@
...
+
+      <div class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
```
Comment (volodymyr-soltys97) [inline src/index.html:24]: Need to use the semantic tag for text, fix it everywhere

File: src/index.html [hunk 4]
```diff
@@ -13,6 +13,40 @@
...
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-title">Price:</div>
+        <div class="card__price-value">$2,199</div>
+      </div>
+      <a
```
Comment (volodymyr-soltys97) [inline src/index.html:43]:
```suggestion
      </div>
      
      <a
```

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,101 @@
...
+  &__price-title {
+    font-size: 12px;
+    color: #616070;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    color: #060b35;
+    font-weight: 500;
+  }
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:76]:
```suggestion
  &__price {
    display: flex;
    position: relative;
    justify-content: space-between;
    align-items: center;
    margin: 0 16px 16px;
    
    &-title {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }

```

File: src/styles/index.scss [hunk 6]
```diff
@@ -1,3 +1,101 @@
...
+
+  &__buy {
+    display: flex;
+    align-self: center;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    width: 166px;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:85]:
```suggestion
    width: 100%;
```

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,3 +1,101 @@
...
+    background-color: #00acdc;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:86]: Create a variable for repeated color and use it everywhere

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

**Comment by Vlad0os13** [comment]
I have a problem with tests, i can't run them localy too because of error(

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls don’t ask for help in github, because this way mentors are only able to see your questions once the review is requested. Instead, pls always ask for help in **fe_chat** if you don't understand something so mentors can quickly help you
So pls ask there for help with tests problems

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Test reports are still not reachable via the link, pls fix and ask for help in chat)

**Pull request closed without merging**